src/Rules/DeadCode/UnreachableStatementRule.php

```diff
@@ -27,7 +28,7 @@ public function processNode(Node $node, Scope $scope): array
 		}
 
 		return [
-			'Unreachable statement - code above always terminates.',
+			RuleErrorBuilder::message('Unreachable statement - code above always terminates.')->line($node->getLine())->build(),
```

Comment:
Hi, this doesn't add any value. If the line() method isn't called on the builder, then $node->getLine() is precisely what is used by default. You can leave RuleErrorBuilder in (since given enough time, I want to all rules return this instance), but remove the line() method call.